### DISCUSSION TITLE
Plotting orientations (EBSD ) map from Euler angle data

### DISCUSSION THREAD

**Discussion by makeabhishek** [mtex-toolbox/mtex#1107] [Ask Anything]
Hi All,
I want to plot grain morphology map from Euler angles. I have a .csv file that contains x coordinate, y coordinate and three euler angles (total 5 columns).

How can I plot the microstructure from this data, similar to the example given in the example, the first image on [https://mtex-toolbox.github.io/ODFTheory.html] .

Regards
Abhishek

**Comment by kilir** [comment]
Hi,

load the data like Ralf explained to you here https://github.com/mtex-toolbox/mtex/discussions/1026, maybe adapt what you need to adapt, then plot it like shown in the documentation.

Cheers,
Rüdiger

**Reply by makeabhishek** [reply]
Thanks for your reply Rudinger, I can plot using the EBSD .ctf file. But is there a way to plot using only Euler angles to plot Euler angles directly?
I can plot those directly in MATLAB using the `imagesc( )` function, I am wondering if there is any function that can do that in MTEX also.

Regards
Abhi

**Reply by kilir** [reply]
It's in the documentation.
https://mtex-toolbox.github.io/EBSDAdvancedMaps.html#2
Rüdiger
